recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/guest-addons.bb

```diff
@@ -65,6 +66,7 @@ RDEPENDS_${PN}-bridge-config = " \
 
 DM_CONFIG_salvator-x-m3-xt = "dm-salvator-x-m3.cfg"
 DM_CONFIG_salvator-x-h3-xt = "dm-salvator-x-h3.cfg"
+DM_CONFIG_salvator-x-m3n-xt = "dm-salvator-x-m3n.cfg"
```

Comment:
Please drop everything related to DM here and everywhere

Reply:
Done